test/cppsuite/configs/search_near_02_stress.txt

```diff
@@ -0,0 +1,31 @@
+# Configuration for search_near_02 stress test.
+# The configuration creates:
+# - threads that continuously insert random keys and values.
+# - threads that continuously perform prefix search_near calls on random keys.
+duration_seconds=1800,
+cache_size_mb=500,
+timestamp_manager=
+(
+    # This will let us randomly pick a read timestamp in a bigger range to trigger visibility
+    # checks.
+    oldest_lag=50,
+),
+workload_generator=
+(
+    populate_config=
+    (
+        collection_count=10,
+    ),
+    insert_config=
+    (
+        key_size=5,
```

Comment:
Thinking we can add a big key size here too? Might add some benefit where key size are really big.

Reply:
I agree. Please @quchenhao, can you check you reproduce WT-7912 with this config too?

Reply:
@etienneptl Just key size?